app/utils/scheduler.py

```
@@ -1537,36 +1541,48 @@ async def _execute_auto_buy(self, send_slack_notification: bool = True):
+                if output3:
+                    # frcr_use_psbl_amt: 외화사용가능금액 (USD)
+                    cash_str = output3.get("frcr_use_psbl_amt") or "0"
+                    try:
+                        available_cash = float(cash_str)
+                        logger.info(f"[{function_name}] 💰 구매 가능 금액 (외화사용가능금액): ${available_cash:,.2f}")
+                        
+                        # 추가 디버깅 정보
+                        frcr_evlu_tota = output3.get("frcr_evlu_tota", "0")
+                        logger.debug(f"[{function_name}] 📊 외화평가총액: ${float(frcr_evlu_tota):,.2f}")
+                    except (ValueError, TypeError) as e:
+                        logger.error(f"[{function_name}] 외화사용가능금액 변환 실패: {cash_str}, 오류: {str(e)}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)